### DISCUSSION TITLE
Pimcore Academy - Which approaches of Web2Print can be implemented with Pimcore?

### DISCUSSION THREAD

**Discussion by alexthunder3** [pimcore/pimcore#7903] [Q&A / Support] [answered]
Hello!
I'm working on the Academy exams and I keep getting stuck on questions that don't seem to be based on the videos or even on the documentation.
Can anyone explain what is a "semi automated print process"? Or what is a "fully automated print process"? Or what is a "creative process"?
I listened to the videos multiple times and neither is mentioned, there is nothing about a semi/fully/creative process in the documentation. I did find a few lines in a marketing article on the pimcore.com website how it "enables the semi-automated and automated creation of print material", but that's it. Answering with "fully" + "semi" is not a valid answer though. So there must be more to it.
It sounds like an official term defined by specific requirements but.. it doesn't seem to be. At least not from what I can tell. I don't have experience with web2print technologies but none of the articles I read on web2print ever mention a fully, semi or creative print process.
I did pass the exam, but I can't say I understand what I selected. I still have no idea what the distinction between them is.

Cheers,
Alex.

**Comment by fashxp** [comment, marked as answer]
Hi,
have a look at the second video of this course: https://pimcore.com/academy/course/Working-with-Documents-Advanced~13475/chapter-13478
There these approaches should be explained. With Pimcore, it is possible to implement all three of them....

BR
Christian

**Reply by alexthunder3** [reply]
Oh, sorry.
Must've been too tired and obsessed on the first longer video.
Thanks!

Regarding the Pimcore Academy videos, I would really love to have speed control and maybe subtitles and/or the transcript. Where can I suggest the feature? Is it okay to open a ticket in this repository, or is there another feedback channel specifically for the Academy?

Best regards,
Alex.

**Reply by fashxp** [reply]
Hi Alex, 
these two things are already on our roadmap. 
In the meanwhile you could use Chrome and an extensions like [Video Speed Control](https://chrome.google.com/webstore/detail/html5-video-speed-control/lcbghbgklnidcfieiicfkjgfokbaeefo?hl=en) to boost or slow down the video speed. 

BR
Christian

**Reply by alexthunder3** [reply]
Hey Christian,
Great, thanks!

I'm not sure where I could post this, but I think I found a typo in the Academy. In the "Working with Objects Advanced" Exam (Senior certificate), the encryption question. One of the answers is "Encrypt data encryption on class definition level.". Shouldn't it be "**Enable** data encryption on class definition level." ?
In the same exam, question 5 "What are possible ways for attaching additional information to relations?", one of the answers "Use field collection with many-to-one relation **ad** additional data attributes." has a typo. 

Cheers and kind regards,
Alex.

**Reply by fashxp** [reply]
Hi Alex,
thank you very much for the hint ... I fixed them. 

BR
Christian
